### DISCUSSION TITLE
Request for Assistance Regarding Flagged Account

### DISCUSSION THREAD

**Discussion by tahereh-s** [community/community#77181] [Other Feature Feedback, Questions, & Ideas]
Dear GitHub Support,
I hope this email finds you well.
My name is tooti and I am writing to you concerning my GitHub account tooti-s which has been flagged. I am unsure as to why this has occurred as I have always strived to adhere to all rules and guidelines set by GitHub.
I have checked my email but have not received any communications explaining why my account has been flagged. I would greatly appreciate any assistance or additional information regarding this situation.
I understand the importance of maintaining a respectful and spam-free environment on GitHub and I assure you that I have not intentionally violated any rules. If there were violations, I would like to know so I can correct them and avoid such mistakes in the future.
I am looking forward to resolving this issue so I can continue contributing to the GitHub community.
Thank you in advance for your time and assistance.
Best Regards,
tooti
[tahereh@craftify.ai](mailto:tahereh@craftify.ai)
tooti-s

**Comment by queenofcorgis** [comment]
Hi @tahereh-s ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) I'm going to close this post as the community is unable to assist.
